frontend/src/container/GridGraphLayout/Graph/FullView/CheckBox.tsx

```diff
@@ -0,0 +1,38 @@
+import { Checkbox, ConfigProvider } from 'antd';
+import { CheckboxChangeEvent } from 'antd/es/checkbox';
+import { ChartData } from 'chart.js';
+
+function CheckBox({
```

Comment:
no need of this component we can directly use the Checkbox from antd

Reply:
I have tried applying the in-line style for dynamically changing the background color and border color for Checkbox In antd also used the `styled` component to change the background color and border color but neither of them worked. Can you suggest any other why to do it?